LiskHQ/lisk-sdk
# Integrate new transaction pool implementation and cleanup - Closes #4856

### What was the problem?

This PR resolves #4856 

### How was it solved?
- Removed old transaction pool implementation
- Integrated new transaction pool implementation
- Update test and code reference
### How was it tested?
- Run the node
- Send the transaction to network and check if its get confirmed
- Start two nodes and sync to see if the transactions are processed as expected
